.github/workflows/ci.yml

```diff
@@ -34,7 +34,7 @@ jobs:
           signingKey: "${{ secrets.CACHIX_SIGNING_KEY }}"
 
       - name: "Run with ${{ matrix.java }} 🚀"
-        run: nix-shell --argstr java "${{ matrix.java }}" --run "sbt ++ 'test;makeSite;doc;it:test'" nix/ci.nix
+        run: nix-shell --argstr java "${{ matrix.java }}" --run "sbt ++ 'it:test;test;makeSite;doc'" nix/ci.nix
```

Comment:
I think it makes sense to run unit tests before integration tests

Reply:
Unfortunately, there's some weirdness in the classloader throwing a `ScalaReflectException` at runtime. Was fixed in https://github.com/cr-org/neutron/pull/189/commits/53811b7ce6e90703fd35da8512f1fed76453f8a5